src/solc_0.7/catalyst/AssetAttributesRegistry.sol

```diff
@@ -135,6 +135,7 @@ contract AssetAttributesRegistry is WithMinter, WithUpgrader {
         uint8 j = 0;
         uint8 i = 0;
         for (i = 0; i < MAX_NUM_GEMS; i++) {
+            // TODO: verify audit comment
             if (j == gemIds.length) {
```

Comment:
generally, the length attribute for dynamic calldata arrays should not be trusted, as it is being encoded in the calldata.
More specifically, the user might submit an array.length that is less than the true value. In this case, it does not seem like an issue, but it would be great to have more people looking on it.

Reply:
This should not be a problem if they provide a shorter length it will be like they provide a shorter array, less gems